Release_Process.md

```diff
@@ -59,3 +59,12 @@ These steps are common between minor and patch releases:
         (without the 'v' in front) is _not_ active.
      3. Make sure the **stable** branch is _not_ active.
      4. Scroll to the bottom of the page and click the Submit button.
+1. Update all deployments with the currently released version accordingly.
```

Comment:
I'd rather not mention anything about updating actual running instances of BigchainDB Server ("deployments" with a lowercase 'd') in the Release Process document. The Release Process is mostly about the steps to make a release (version X.Y.Z) of BigchainDB Server. We could have a whole other document about how to update a running BigchainDB Server instance. Indeed, that's partly what this page is about: https://docs.bigchaindb.com/projects/server/en/latest/cloud-deployment-templates/upgrade-on-kubernetes.html